### DISCUSSION TITLE
DevOps and Build tools

### DISCUSSION THREAD

**Discussion by shenoy-anurag** [shenoy-anurag/simple-reddit#239] [Ideas]
I'm thinking of containerizing the backend, frontend and the database. I plan to use docker-compose to allow us to spin up the entire ecosystem for quicker development. Similar to what I've done in this repo: <https://github.com/shenoy-anurag/Aggre-Gator>.

What do you think?

**Comment by Ganesh96** [comment]
Okay, can you include me in while doing that?

**Reply by shenoy-anurag** [reply]
Yes, definitely. I'll be a bit busy for the next couple of days, so maybe after Thursday or the coming weekend will be a good time.

**Comment by shenoy-anurag** [comment]
Do go through this tutorial series until we work on the dockerization together.
<https://www.digitalocean.com/community/tutorials/the-docker-ecosystem-an-introduction-to-common-components>

**Comment by shenoy-anurag** [comment]
Also download and install docker and docker-compose on your machine.
<https://docs.docker.com/desktop/windows/install/>
